src/adapter/adapter.js

```diff
@@ -0,0 +1,222 @@
+// @flow
+import youbora from 'youboralib'
+import { Error } from 'playkit-js'
+import { MediaType } from 'playkit-js'
+
+let YouboraAdapter = youbora.StandardAdapter.extend({
+
+  constructor: function (player, config) {
+    this.config = config
+    YouboraAdapter.__super__.constructor.call(this, player)
+  },
+
+  /**  @returns {String} - current plugin version */
+  getVersion: function () {
+    return youbora.VERSION + '-' + __VERSION__ + '-' + __NAME__
+  },
+
+  /**  @returns {Number} - current playhead of the video */
+  getPlayhead: function () {
+    return this.player.currentTime
+  },
+
+  /**  @returns {Number} - current playrate */
+  getPlayrate: function () {
+    return this.player.playbackRate
+  },
+
+  /**  @returns {Number} - video duration */
+  getDuration: function () {
+    return this.player.duration || null
+  },
+
+  /**  @returns {Number} - current bitrate */
+  getBitrate: function () {
+    let activeVideo = this.player.getActiveTracks().video
+    if (activeVideo && activeVideo.bandwidth) {
+      return isNaN(activeVideo.bandwidth) ? null : activeVideo.bandwidth
+    }
+    return null
+  },
+
+  /**  @returns {String} - rendition */
+  getRendition: function () {
+    let activeVideo = this.player.getActiveTracks().video
+    if (activeVideo) {
+      if (isNaN(activeVideo.bandwidth)) {
+        return null
+      }
+      return youbora.Util.buildRenditionString(activeVideo.width, activeVideo.height, activeVideo.bandwidth)
+    }
+    return null
+  },
+
+  /**  @returns {String} - title */
+  getTitle: function () {
+    return this.config.entryName
+  },
+
+  /**  @returns {Boolean} - true if live and false if VOD */
+  getIsLive: function () {
+    return this.config.entryType === MediaType.LIVE
```

Comment:
@jesuslc91 this is a bug, you should call `player.isLive()` to know if media is live